src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -140,27 +229,46 @@ public void toModelType_invalidAddress_throwsIllegalValueException() {
     public void toModelType_invalidClientSources_throwsIllegalValueException() {
         List<JsonAdaptedClientSource> invalidClientSources = new ArrayList<>(VALID_CLIENTSOURCES);
         invalidClientSources.add(new JsonAdaptedClientSource(INVALID_CLIENTSOURCE));
-        JsonAdaptedPerson person =
-                new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL,
-                        VALID_ADDRESS, invalidClientSources, VALID_NOTE, VALID_IS_ARCHIVE, VALID_PRIORITY);
+        JsonAdaptedPerson person = new JsonAdaptedPerson(
+                VALID_NAME,
+                VALID_PHONE,
+                VALID_EMAIL,
+                VALID_ADDRESS,
+                invalidClientSources,
+                VALID_NOTE,
+                VALID_IS_ARCHIVE,
+                VALID_PRIORITY,
+                new JsonAdaptedPolicy(VALID_POLICY_NAME, VALID_POLICY_DESCRIPTION));
         assertThrows(IllegalValueException.class, person::toModelType);
     }
 
     @Test
     public void toModelType_invalidNote_throwsIllegalValueException() {
-        JsonAdaptedPerson person =
-                new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL,
-                        VALID_ADDRESS, VALID_CLIENTSOURCES, INVALID_NOTE, VALID_IS_ARCHIVE, VALID_PRIORITY);
+        JsonAdaptedPerson person = new JsonAdaptedPerson(
+                VALID_NAME,
+                VALID_PHONE,
+                VALID_EMAIL,
+                VALID_ADDRESS,
+                VALID_CLIENTSOURCES,
+                INVALID_NOTE,
+                VALID_IS_ARCHIVE,
+                VALID_PRIORITY,
+                new JsonAdaptedPolicy(VALID_POLICY_NAME, VALID_POLICY_DESCRIPTION));
         assertThrows(IllegalValueException.class, person::toModelType);
     }
 
     @Test
     public void toModelType_invalidPriority_throwsIllegalValueException() {
-        JsonAdaptedPerson person =
-                new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL,
-                        VALID_ADDRESS, VALID_CLIENTSOURCES, VALID_NOTE, VALID_IS_ARCHIVE, INVALID_PRIORITY);
+        JsonAdaptedPerson person = new JsonAdaptedPerson(
+                VALID_NAME,
+                VALID_PHONE,
+                VALID_EMAIL,
+                VALID_ADDRESS,
+                VALID_CLIENTSOURCES,
+                VALID_NOTE,
+                VALID_IS_ARCHIVE,
+                INVALID_PRIORITY,
+                new JsonAdaptedPolicy(VALID_POLICY_NAME, VALID_POLICY_DESCRIPTION));
         assertThrows(IllegalValueException.class, person::toModelType);
     }
```

Comment:
Perhaps we should have invalidPolicy test, with both invalid name and description